core/src/main/scala/io/projectglow/vcf/VCFInputFormatter.scala

```diff
@@ -48,11 +48,7 @@ class VCFInputFormatter(
   }
 
   override def write(record: InternalRow): Unit = {
-    val vcOpt = converter.convert(record)
-    if (vcOpt.isDefined) {
-      val vc = vcOpt.get
-      writer.write(vc)
-    }
+    converter.convert(record).foreach(writer.write)
```

Comment:
No performance penalty for the lambda function here?

Reply:
I don't think it's an issue for `Option`s; the main perf issues are for Lists/Maps/etc.